Cargo.toml

```diff
@@ -18,6 +18,7 @@ serde_derive = "1.0.114"
 reqwest = { version = "0.11.0", features = ["json"] }
 ethers = "1.0.0"
 regex = "1.7.1"
+lazy_static = "1.4.0"
```

Comment:
Can we use `once_cell` for all global static values? Seems redundant to have both crates